examples/petsc/index.rst

```diff
@@ -138,8 +138,8 @@ and thus the weak Laplacian may be expressed as
 .. math::
    :label: eq-weak-laplace-sphere
 
-   \int_S \frac{\partial v}{\partial \overset\circ{\mathbf x}} \left( \frac{\partial u}{\partial \overset\circ{\mathbf x}} \right)^T
-       = \int_S \frac{\partial v}{\partial \mathbf X} \underbrace{\frac{\partial \mathbf X}{\partial \overset\circ{\mathbf x}} \left( \frac{\partial \mathbf X}{\partial \overset\circ{\mathbf x}} \right)^T}_{\mathbf g_{(2\times 2)}}  \left(\frac{\partial u}{\partial \mathbf X} \right)^T
+   \int_{\Omega} \frac{\partial v}{\partial \overset\circ{\mathbf x}} \left( \frac{\partial u}{\partial \overset\circ{\mathbf x}} \right)^T
```

Comment:
I took the occasion to homogenize the volume/surface integrals notation in all our examples docs. We had mainly used `dV` and `dS` in all other examples, including the NS mini-app.

@jedbrown, here in the BPs on the cubed-sphere you had used `S`. I guess it was the domain `\Omega` in this case. Now this equation and the one below are the only ones that are missing the differentials. I wanted to make sure I understood your notation. We should add them for consistency

Reply:
For the sphere, we should perhaps use `\int_{S^2}` in place of `\int_{\Omega}` to make clear that the domain is the surface of the 2-sphere.

The differential is always with respect to the volume of integration, and is thus redundant information if we always specify the domain of integration. One could use `\oint_{S^2}` as a more explicit alternative.

Reply:
I prefer keeping consistency with all other examples. I have clarified what `\Omega` means in this case.

Reply:
Okay, thanks for the consistency. At least we're using `\diff` for upright `d` for functional differentials, versus standard `d` for volume measures.